server/runtime/src/main/java/org/infinispan/server/configuration/security/LdapUserPasswordMapperConfiguration.java

```diff
@@ -11,7 +11,7 @@
  * @since 10.0
  */
 public class LdapUserPasswordMapperConfiguration extends ConfigurationElement<LdapUserPasswordMapperConfiguration> {
-   static final AttributeDefinition<String> FROM = AttributeDefinition.builder(Attribute.FROM, "userPassword", String.class).immutable().build();
+   static final AttributeDefinition<String> FROM = AttributeDefinition.builder(Attribute.FROM, null, String.class).immutable().build();
```

Comment:
Leave the default as-is. The only alternative that I know of is Active Directory, which uses `sAMAccountName`, but you cannot use that for evidence verification (only direct verification is possible).